apps/assistants/models.py

```diff
@@ -64,8 +64,6 @@ class OpenAiAssistant(BaseTeamModel, VersionsMixin):
     is_archived = models.BooleanField(default=False)
     tools = ArrayField(models.CharField(max_length=128), default=list, blank=True)
 
-    files = models.ManyToManyField("files.File", blank=True)
```

Comment:
This is a legacy field which is no longer used. I checked on prod and there are no assistants with any files linked through this field.